ui.content/src/main/content/jcr_root/conf/acm/settings/snippet/available/core/argument/path.yml

```diff
@@ -0,0 +1,15 @@
+group: argument
+name: argument_path
+content: |
+  args.path("${1:name}") { label = "${2:label}"; root = "${4:value}" }
+documentation: |
+  An argument that represents a repository path with a path picker dialog.
```

Comment:
arguments is not represented by ... but allows to choose value by picker ; or sth like that / see other args

Reply:
make the whole doc more strict & concise please